app/styles/ember-frost-table/_frost-table.scss

```diff
@@ -20,6 +35,10 @@
   }
 
   &-header {
+    display: flex;
+    flex: none;
+    flex-direction: row;
+    justify-content: flex-start;
```

Comment:
Probably no need to use flex here at all, you're going to set the header column widths programatically so this can remain as a table row